Chatkit/Auth/TokenProvider.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  TokenProvider.swift
+//  Chatkit
+//
+//  Created by Mike Pye on 06/12/2019.
```

Comment:
What's the motivation for having "created by" comment? It feels like it's being used less and less often in the industry. After all a better info can be found in VCS.

Reply:
XCode adds it by default, I can delete it